internal/config/main.go

```diff
@@ -126,12 +129,18 @@ func (c *Config) AutoDetect() error {
 		}
 	}
 
-	hpaVersion, err := c.autoDetect.HPAVersion()
+	hpaV, err := c.autoDetect.HPAVersion()
 	if err != nil {
 		return err
 	}
-	c.autoscalingVersion = hpaVersion
-	c.logger.V(2).Info("autoscaling version detected", "autoscaling-version", c.autoscalingVersion.String())
+	if c.hpaVersion.Get() != hpaV {
+		c.logger.V(1).Info("HPA version detected", "version", hpaV)
```

Comment:
Why are we changing the log level here?

Reply:
I set the same value used for the OpenShift Routes:
```go
		c.logger.V(1).Info("openshift routes detected", "available", ora)
```

I can keep it, just revert it or revert it + change the value for the OpenShift Routes. What do you think?